app/src/features/surveys/view/survey-animals/SurveyAnimalsTable.tsx

```diff
@@ -47,17 +50,17 @@ export const SurveyAnimalsTable = ({
     : animalData;
 
   const columns: GridColDef<ISurveyAnimalsTableEntry>[] = [
-    {
-      field: 'critter_id',
-      headerName: 'Critter ID',
-      flex: 1,
-      minWidth: 300
-    },
     {
       field: 'animal_id',
-      headerName: 'Animal ID',
+      headerName: 'Alias',
       flex: 1
     },
+    {
+      field: 'wlh_id',
+      headerName: 'WLH ID',
+      flex: 1,
+      renderCell: (params) => <Typography>{params.value ? params.value : 'None'}</Typography>
```

Comment:
Can just coalesce this: `params.value ?? 'None'`

Reply:
Doing this will also display None if this value happens to be empty string. I think that's desirable.

Reply:
Ahh you're right. I think I meant to say `params.value || 'None'`, which is like a falsy coalesce rather than `??` which is a null coalesce... I think?

Reply:
^ correct